design/one-pager-ignore-changes.md

```diff
@@ -77,44 +77,44 @@ This will allow users to fine-tune how Crossplane manages the external
 resource, in a manner which is very explicit and easy to understand.
 
 Some examples on how the management policies would work and how they would
-replace the current `managementPolicy` and `deletionPolicy`:
+replace the current `ManagementPolicies` and `deletionPolicy`:
 
 ```yaml
 # Default
 spec:
-  managementPolicy: FullControl
+  ManagementPolicies: FullControl
```

Comment:
```suggestion
  managementPolicies: FullControl
```

I think all of these should be `managementPolicies` with a lower case m.

Reply:
Ahh right 🤦 updated with lowercase.